ORStools/common/directions_core.py

```diff
@@ -117,8 +117,13 @@ def get_fields(
 
 
 def get_output_feature_directions(
-    response, profile, preference, options=None, from_value=None, to_value=None
-):
+    response: dict,
+    profile: str,
+    preference: str,
+    options: str = None,
```

Comment:
```suggestion
    options: Optional[str] = None,
```
If None is the default value, it implies it's a valid value for the argument, so the type should be optional